### PR TITLE
enforce the correct reset on the NanoAODs for every event loop

### PR DESCRIPTION
**Pull request by mazimm**
add dummy arrays to the NanoAODArrayMaker so that the analysis arrays will be reseted for every event

### CODE REVIEW

**Comment by raymondEhlers** [comment]
+test

File: PWG/DevNanoAOD/AliAnalysisNanoAODCuts.cxx [hunk 1]
```diff
@@ -120,13 +120,17 @@ void AliNanoAODSimpleSetter::SetNanoAODHeader(const AliAODEvent * event   , AliN
 
   TString firedTriggerClasses = header->GetFiredTriggerClasses();
 
-  static const char * validatorString[] = {"Centr","CentrTRK","CentrCL0","CentrCL1", "MagField", "OfflineTrigger", "RunNumber", "FiredTriggerClasses", 0};
+  UShort_t bunchCrossNumber = header->GetBunchCrossNumber();
+  UInt_t orbitNumber = header->GetOrbitNumber();
+  UInt_t periodNumber = header->GetPeriodNumber();
+
+  static const char * validatorString[] = {"Centr","CentrTRK","CentrCL0","CentrCL1", "MagField", "OfflineTrigger", "RunNumber", "FiredTriggerClasses", "BunchCrossNumber", "OrbitNumber", "PeriodNumber", 0};
```
Comment (mfasDa) [inline PWG/DevNanoAOD/AliAnalysisNanoAODCuts.cxx]: Why static, and why const char *? Better use array of const TStrings (in particular const std::array< const TString, ...>), non-static.

**Comment by mfasDa** [comment]
+test

**Comment by raymondEhlers** [comment]
It seems they didn't fix the test + test = +1 bug...

**Pull request merged**